### ISSUE TITLE
Fix background-color menu items to improve a11y

### ISSUE THREAD

**Issue by MinThaMie** [empress/field-guide-default-template#37]
The `#eee` background color for hover and active items results in failing a11y contrast checks. Could be fixed by picking a lighter gray.
